spec/jobs/replication/integration_spec.rb

```diff
@@ -23,13 +23,6 @@
   let(:ibm_provider) { instance_double(Replication::IbmProvider, bucket: ibm_bucket) }
   let(:moab_storage_root) { MoabStorageRoot.find_by!(name: 'fixture_sr1') }
 
-  around do |example|
-    old_adapter = ApplicationJob.queue_adapter
-    ApplicationJob.queue_adapter = :inline
-    example.run
-    ApplicationJob.queue_adapter = old_adapter
-  end
```

Comment:
No adapter switcheroo shenanigans are needed if we lean on Rails' built-in enqueued jobs helper.

Reply:
fwiw, my memory is that we did this to make tests more reliable.  but my memory is also that we did this right around when we were finishing up replication and replication audit, at the end of the first set of work cycles.  and of course the job coordination and job de-duplication were just a little shakier then.  so it's possible that this was actually papering over small issues that've been fixed over the years.  anyway, more than happy to drop this and see if it works out.